### ISSUE TITLE
Accessibility: avoid non-link links

### ISSUE THREAD

**Issue by afercia** [swissspidy/oEmbed-API#43]
Noticed a couple of `a href="#"` for the sharing dialog open/close. They're not real links, they don't bring you to any resource. They're UI controls that "do" something when activated and should be preferably be button elements.
